weave/panels_py/instructions/panel_llm_monitor.md

```diff
@@ -1,23 +1,38 @@
-Simply set the OpenAI API Base to `https://api.wandb.ai/proxy/openai/v1` and the OpenAI API Key to the concatenation of your W&B API Key ([wandb.ai/authorize](https://wandb.ai/authorize)) and current OpenAI API Key ([platform.openai.com/account/api-keys](https://platform.openai.com/account/api-keys)) in order to track all API calls! Please refer to the [Weave Monitoring README](https://github.com/wandb/weave/tree/master/examples/monitoring) for full details. Features:
+## OpenAI Monitoring Quickstart
 
-- Automatically track LLM usage and aggregate useful metrics like cost, latency and throughput across your projects/teams
-- Dynamically query and derive insights from the logs of all your OpenAI API calls
-- Iterate visually to slice, aggregate, and explore your data
+To start monitoring your OpenAI calls:
 
-### Setup with environment variables
+- Set the OpenAI API key to the concatenation of your [W&B API key](https://wandb.ai/authorize) and [OpenAI API key](https://platform.openai.com/account/api-keys).
+- Set the OpenAI API base URL to `https://api.wandb.ai/proxy/openai/v1`.
+
+### Step 1: Set environment variables
 
 ```shell
-OPENAI_API_BASE="https://api.wandb.ai/proxy/openai/v1"
-OPENAI_API_KEY="$WANDB_API_KEY:$OPENAI_API_KEY"
+export WANDB_API_KEY="" # W&B API Key from https://wandb.ai/authorize
+export WANDB_OPENAI_API_KEY="$WANDB_API_KEY:$OPENAI_API_KEY"
 ```
 
-### Setup with Python library
-
-[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://github.com/wandb/weave/blob/master/examples/monitoring/openai_proxy_quickstart.ipynb)
+### Step 2: Set local variables in your code
 
 ```python
+import os
 import openai
 openai.api_base = "https://api.wandb.ai/proxy/openai/v1"
-openai.api_key = f"{WANDB_API_KEY}:{OPENAI_API_KEY}"
-openai.ChatCompletion.create(...)
+openai.api_key = os.getenv("WANDB_OPENAI_API_KEY")
 ```
+
+### Step 3: Create a monitoring board
+
+Use OpenAI as normal and navigate to `monitoring/openai` via the project browser on the left to create a board to visualize your usage.
+
+## In-depth Tutorials
+
+For more ways to configure monitoring and understand LLM usage, follow along with one of our tutorials:
+
+| Method | Tutorial | Use Case | 
+|--------|----------|---------------|
+| Python Client |[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/wandb/weave/blob/master/examples/monitoring/openai_client_quickstart.ipynb)| For devs, visualize, understand, and customize analysis of your LLMs | 
+| OpenAI Proxy | [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/wandb/weave/blob/master/examples/monitoring/openai_proxy_quickstart.ipynb) | For teams, track LLM usage and key metrics like cost/latency across projects |
```

Comment:
can we put the proxy one first, since that is what we are saying to use in the code blocks above?